### PR TITLE
PLAT-517 add service-workflow skeleton

### PR DESCRIPTION
**Pull request by niishiikar**
**1. Whats changing**

Service workflow skeleton added

**2. Description**

Added the skeleton of the service to begin working.

**3. Jira**

https://mindstronghealth.atlassian.net/browse/PLAT-517

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+const express = require('express');
+const package_json = require('../package.json');
+require('dotenv').config();
+
+// express app
+const app = express();
+
+checkToEnforceSecurityHeaders(app);
+
+app.use(ifnotmatch('/v1/callback', bodyParser.urlencoded({extended: true})));
```
Comment (rajeevc) [inline src/index.ts:43]: match rules parts we may not need - for acuity?

File: src/tools/authHelper.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+const dotenvExpand = require('dotenv-expand');
+const myenv = require('dotenv').config();
+dotenvExpand(myenv);
+
+const AUTH_URL = process.env.authorizationUrl || 'FILLIN';
```
Comment (rajeevc) [inline src/tools/authHelper.ts:5]: do we need this file?

File: buildsvc.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#build
+npm install
+npm run build
+
+#WEST:
+# shellcheck disable=SC2046
+eval $(aws ecr get-login --no-include-email --region us-west-2)
+docker build -t serviceworkflow .
+docker tag serviceworkflow:latest 524422562296.dkr.ecr.us-west-2.amazonaws.com/service-workflow:latest
+docker tag serviceworkflow:latest 524422562296.dkr.ecr.us-west-2.amazonaws.com/service-workflow:"${SVC_VERSION}"
```
Comment (pmogilev) [inline buildsvc.sh:29]: please add docker tagging for a branch specific names

**Comment by pmogilev** [comment]
@niishiikar did you set permission for this project for mindstorng team?

File: buildsvc.sh (hunk 3 above)
Comment (niishiikar) [reply buildsvc.sh:29]: added

File: src/index.ts (hunk 1 above)
Comment (niishiikar) [reply src/index.ts]: changed.

File: src/tools/authHelper.ts (hunk 2 above)
Comment (niishiikar) [reply src/tools/authHelper.ts]: removed

**Comment by niishiikar** [comment]
@pmogilev added mindstrong team to repo.

**Pull request merged**